.github/workflows/tests.yml

```diff
@@ -0,0 +1,51 @@
+name: tests-pipelinedp
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  linting:
+
+    runs-on: ubuntu-latest
```

Comment:
is it possible to run on Windows and Mac as well?

Reply:
Linting tests are not required to be run on all OS. One should suffice. We can run test cases on all OS.